build_modules/test/build_test.dart

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
```

Comment:
I expected this to just be a move, we should be able to delete this equivalent test in build_web_compilers (and its dependencies on json_serializable, source_gen, and json_annotation)